src/main/webapp/creatives/maps/simid_map.css

```diff
@@ -40,10 +70,11 @@ html, body {
  */
 #map {
     height: 80%;
-    width: 80%;
+    width: 100%;
     margin: auto;
     position: relative;
     top: 215;
     -ms-transform: translateY(-50%);
     transform: translateY(-50%);
+    overflow: inherit !important;
```

Comment:
I tried, but couldn't find a way

Reply:
Can you try making a more specific css rule to override it?
https://www.w3schools.com/css/css_specificity.asp

Reply:
The styling for the map isn't 100% correct as of now because I know Kristen was working on that too. Once we have all of our css for the map together, I can try and work on figuring out if there's a way to specify this further.